solr/solrj/src/java/org/apache/solr/client/solrj/io/stream/metrics/MaxMetric.java

```diff
@@ -86,7 +86,7 @@ public void update(Tuple tuple) {
       if(l > longMax) {
         longMax = l;
       }
-    } else {
+    } else if(o instanceof Long) {
```

Comment:
```suggestion
    } else if (o instanceof Long) {
```

Reply:
the current format is consistent with the rest of the source file, best not to mix formatting even if it's wrong